src/modules/core/components/ArgumentDisplay/ArgumentDisplay.tsx

```diff
@@ -0,0 +1,67 @@
+import React from 'react';
+
+import HookedUserAvatar from '~users/HookedUserAvatar';
+import TimeRelative from '~core/TimeRelative';
+import Tag from '~core/Tag';
+import { useUser } from '~data/index';
+import { getMainClasses } from '~utils/css';
+
+import styles from './ArgumentDisplay.css';
+
+interface Appearance {
+  theme?: 'motion' | 'objection';
+}
+
+interface Argument {
+  author: string;
+  comment: string;
+  timestamp: number;
+}
+
+interface Props {
+  appearance?: Appearance;
+  argument: Argument;
+}
+
+const displayName = 'ArgumentDisplay';
+
+const ArgumentDisplay = ({
+  appearance = { theme: 'motion' },
+  argument: { author, timestamp, comment },
+}: Props) => {
+  const user = useUser(author);
+  const tagTheme = appearance.theme === 'motion' ? 'primary' : 'pink';
+  const tagText = appearance.theme === 'motion' ? 'Motion' : 'Objection';
+  const UserAvatar = HookedUserAvatar({ fetchUser: false });
+
+  return (
+    <div
+      className={`${styles.container} ${getMainClasses(appearance, styles)}`}
```

Comment:
i think if you create .main class in styles file and apply container styles, then here you could just do: 
```suggestion
      className={getMainClasses(appearance, styles)}
```

Reply:
Not working, from what I saw inside `getMainClasses` I think it only works like that when no `theme` is provided